app/bundles/LeadBundle/Entity/CustomFieldRepositoryTrait.php

```diff
@@ -125,7 +125,8 @@ public function getEntitiesWithCustomFields($object, $args, $resultsCallback = n
 
                 //ORM - generates lead entities
                 /** @var \Doctrine\ORM\QueryBuilder $q */
-                $q = $this->getEntitiesOrmQueryBuilder($order);
+                $joinIpAddresses = isset($args['joinIpAddresses']) ? $args['joinIpAddresses'] : true;
+                $q               = $this->getEntitiesOrmQueryBuilder($order, $joinIpAddresses);
```

Comment:
I was doing similar change in https://github.com/mautic/mautic/pull/11842. How about to do just
```suggestion
                $q = $this->getEntitiesOrmQueryBuilder($order, $args);
```
on this line.

Reply:
Thanks @escopecz, I used this apporach